coderd/workspaceapps/token.go

```diff
@@ -0,0 +1,206 @@
+package workspaceapps
```

Comment:
This entire file is really clean and well documented 😍😍😍

Reply:
Unfortunately we will have to redo a bit of it because we are now signing and encrypting with the same key, which is a "no-no" in security :cry:.

Probably just going to go full encryption and use JWEs.

Reply:
I changed the key length to be 96 bytes. We use the first 64 bytes as the signing key and the last 32 bytes as the encryption key for API keys.